src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -446,6 +446,87 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+  #
+  # Get an Anonymous Elector Document (AED) from its application identifier
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-elector-documents/applications/{applicationId}':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application.
+        schema:
+          type: string
+        in: path
+        required: true
+      - name: applicationId
+        description: The identifier of the Anonymous Elector Document.
+        schema:
+          type: string
+        in: path
+        required: true
+    options:
+      summary: CORS support
+      description: |
+        Enable CORS by returning correct headers
+      tags:
+        - Anonymous Elector Documents (AED)
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    get:
+      summary: Returns the Anonymous Elector Document summary for an application
+      description: Returns the Anonymous Elector Document summary for an application
+      tags:
+        - Anonymous Elector Documents (AED)
+      responses:
+        '200':
+          $ref: '#/components/responses/AnonymousElectorDocumentSummaries'
+        '404':
+          description: Not Found
+      operationId: get-anonymous-elector-document-summary-by-application-id
+      security:
+        - eroUserCognitoUserPoolAuthorizer: [ ]
+      x-amazon-apigateway-integration:
+        type: HTTP_PROXY
+        uri: '${base_uri}/eros/{eroId}/anonymous-elector-documents/applications/{applicationId}'
```

Comment:
To contradict myself slightly here, I think `/applications` is ok here as it's clearly in the path/context of `/anonymous-elector-documents`.